app/core/stt_test.py

```
@@ -0,0 +1,198 @@
+
+                    if t == "stt_interim" and t_first_interim is None:
+                        t_first_interim = now
+
+                    elif t == "stt_final":
+                        stt_text    = msg.get("text", "")
+                        t_stt_final = now
+
+                    elif t in ("done", "error"):
+                        break
+            except Exception:
+                pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)